packages/table/tsconfig.json

```diff
@@ -3,6 +3,17 @@
     "declaration": true,
     "experimentalDecorators": true,
     "jsx": "react",
+    "lib": [
+      "dom",
+      "es2015.core",
+      "es2015.collection",
+      "es2015.generator",
+      "es2015.iterable",
+      "es2015.promise",
+      "es2015.reflect",
+      "es2015.symbol",
```

Comment:
yup, it's also the reason we get some errors in the build from unused variables (see below)